kratos/processes/variational_distance_calculation_process.h

```diff
@@ -357,15 +355,11 @@ class VariationalDistanceCalculationProcess : public Process
 
     virtual void Clear()
     {
-        Model& current_model = mr_base_model_part.GetModel();
-        ModelPart& r_distance_model_part = current_model.GetModelPart( mAuxModelPartName );
-        r_distance_model_part.Nodes().clear();
-        r_distance_model_part.Conditions().clear();
-        r_distance_model_part.Elements().clear();
-        // r_distance_model_part.GetProcessInfo().clear();
-        mdistance_part_is_initialized = false;
+        if(mrModel.HasModelPart( mAuxModelPartName ))
+            mrModel.DeleteModelPart( mAuxModelPartName );
```

Comment:
Is this correct?

Reply:
I was thinking on `mrAuxModelPart.clear()` (note that this implies having a reference to `AuxModelPart`.